pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -426,7 +426,7 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 			Alert: "KubeVirtComponentExceedsRequestedMemory",
 			Expr: intstr.FromString(
 				// In 'container_memory_working_set_bytes', 'container=""' filters the accumulated metric for the pod slice to measure total Memory usage for all containers within the pod
-				fmt.Sprintf(`((kube_pod_container_resource_requests{namespace="%s",container=~"virt-controller|virt-api|virt-handler|virt-operator",resource="memory"}) - on(pod) group_left(node) container_memory_working_set_bytes{container="",namespace="%s"}) < 0`, ns, ns)),
+				fmt.Sprintf(`((kube_pod_container_resource_requests{namespace="%s",container=~"virt-controller|virt-api|virt-handler|virt-operator",resource="memory"}) - on(pod) group_left(node) container_memory_working_set_bytes{container=~"virt-controller|virt-api|virt-handler|virt-operator",namespace="%s"}) < 0`, ns, ns)),
```

Comment:
Does this mean we would only track container that are named `virt-contoller`/`virt-launcher`, etc?

If so, please take into account that some of our Pods contain multiple (a.k.a. side-car) containers. Does this mean that the alert would fire for these containers?

Reply:
I guess you are talking abut another question: 

Even for left hand, is current container filter enough? `https://github.com/kubernetes/kube-state-metrics/blob/main/docs/pod-metrics.md` . for those side-car, it may not be enough.

The fix, is for another issue, the left-hand result can not meet equally with the right-hand result.